.env.example

```
@@ -77,4 +77,9 @@ MEDIA_DISK=
 FILAMENT_FILESYSTEM_DISK=
 
 NIGHTWATCH_TOKEN=
-NIGHTWATCH_REQUEST_SAMPLE_RATE=0.1
\ No newline at end of file
+NIGHTWATCH_REQUEST_SAMPLE_RATE=0.1
+
+MISTRAL_API_KEY=
+MISTRAL_MODEL=mistral-small-latest
+MISTRAL_MAX_TOKENS=2000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MISTRAL_MAX_TOKENS key should go before the MISTRAL_MODEL key

(UnorderedKey)